src/components/v2/V2Project/V2FundingCycleSection/index.tsx

```diff
@@ -59,22 +79,23 @@ export default function V2FundingCycleSection({
     )
   }
 
+  const fundingCycleData = serializeV2FundingCycleData(fundingCycle)
+
   const tabs = [
     {
       key: 'current',
       label: tabText({ text: t`Current` }),
       content: <CurrentFundingCycle expandCard={expandCard} />,
     },
-    !isPreviewMode && {
-      key: 'upcoming',
-      label: tabText({ text: t`Upcoming` }),
-      content: <UpcomingFundingCycle expandCard={expandCard} />,
-    },
+    !isPreviewMode &&
+      fundingCycle &&
```

Comment:
`fundingCycle` is already guarded against on line 50, i thinkwe can remove this check here

Reply:
Yup good catch